packages/docs/content/06-integrations/00-metallb.md

```diff
@@ -1,45 +1,60 @@
----
-title: 'MetalLB'
-metaTitle: 'MetalLB'
-metaDescription: 'MetalLB Load Balancer pack in Spectro Cloud'
-hiddenFromNav: true
-isIntegration: true
-category: ['load balancers']
-logoUrl: 'https://registry.spectrocloud.com/v1/lb-metallb/blobs/sha256:3d09a1eab856a03d5b821062dcd1da624256e8f1e2ede404d88cb088d3adb945?type=image/png'
----
-
+---
+title: 'MetalLB'
+metaTitle: 'MetalLB'
+metaDescription: 'MetalLB Load Balancer pack in Spectro Cloud'
+hiddenFromNav: true
+isIntegration: true
+category: ['load balancers']
+logoUrl: 'https://registry.spectrocloud.com/v1/lb-metallb/blobs/sha256:3d09a1eab856a03d5b821062dcd1da624256e8f1e2ede404d88cb088d3adb945?type=image/png'
+---
+
 import Tabs from '@librarium/shared/src/components/ui/Tabs';
 import WarningBox from '@librarium/shared/src/components/WarningBox';
 import InfoBox from '@librarium/shared/src/components/InfoBox';
 import PointsOfInterest from '@librarium/shared/src/components/common/PointOfInterest';
 import Tooltip from "@librarium/shared/src/components/ui/Tooltip";
-
-
-# MetalLB
-
-MetalLB is a load-balancer implementation for bare metal [Kubernetes](https://kubernetes.io/) clusters, using standard routing protocols. This integration is recommended for the on-prem cloud(s) and will help external service(s) get an IP address when the service type is set as LoadBalancer.
-
-## Versions Supported
+
+
+# MetalLB
+
+MetalLB is a load-balancer implementation for bare metal [Kubernetes](https://kubernetes.io/) clusters, using standard routing protocols. This integration is recommended for the on-prem cloud(s) and will help external service(s) get an IP address when the service type is set as LoadBalancer.
+
+## MetalLB Pack Working Details:
+
+* The address set in pack values goes into a configMap config in metallb-system namespace. This configMap is used by the MetalLB controller and speakers as volume mounts.

```

Comment:
config is the name of the configmap. Likewise, metallb-system is the default namespace used for MetalLB installation. Can we highlight them as code so that they'll stand-out among other words. Examples of this can be found in https://docs.spectrocloud.com/integrations/kubernetes-dashboard/